src/_pytest/debugging.py

```diff
@@ -11,6 +12,31 @@
 from _pytest.config import hookimpl
 
 
+def _validate_usepdb_cls(value):
```

Comment:
I think this is an excellent candidate for unittests instead of integration tests... what do you think?

Reply:
Ok, there is one unit test already, will move one of the integration tests there also.